### ISSUE TITLE
Antidot does not creaate fish env/alias files

### ISSUE THREAD

**Issue by NoelJacob** [doron-cohen/antidot#225]

**Comment by doron-cohen** [comment]
Hey @NoelJacob . Could you elaborate on the steps you took and what was the effect? It's really hard to know what happened with such a short title.

**Comment by NoelJacob** [comment]
I did the exact proper usage and I only got bash files generated in the config folder

**Comment by doron-cohen** [comment]
Can you try running `antidot init -s fish`? I get this issue too. Working on a fix.

**Comment by doron-cohen** [comment]
The issue for me was that `echo $SHELL` was returning `/bin/zsh` instead of `/usr/local/bin/fish`. I believe this is caused since the terminal was running inside VSCode and it somehow messed up the environment variables.

**Comment by NoelJacob** [comment]
Mine was running in a normal fish terminal. Also if it had assumed the shell was zsh shouldn't it produce zsh files from json? Mine only created the default bash file. Same with and without -s fish flag.

**Comment by doron-cohen** [comment]
Yes I know of these. I use fish and I enjoy it very much. I am working on changing the way antidot works. Hope it will make this issue irrelevant.

**Comment by knofte** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://link.bnbreward.online/0mhPJ2)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](http://link.bnbreward.online/0mhPJ2) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.500k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @jyeung205, @kevinyuan, @wisbech, @brodock, @yourbossisnotmyboss, @riboot, @MrMoshkovitz, @fauwzk, @larry-lime, @tallstoat, @jeliasson, @tenpaiyomi, @jeffwalter-rum, @Wi11iamC, @AnakDagang, @nps1ngh, @SnowpMakes, @yellowcandle, @Skyblueballykid, @Saptarshi-max, @EddieOlson, @apsifly, @denvera, @gong-chengxiao, @Stevepiquito, @thedrawing, @liuxiong21, @dimacros, @guyernest, @lgqfhwy, @MohsenRazvi, @Banguiskode, @glows, @the-schmitt, @evan-duncan, @tedffx2, @J3rome, @eduardomdrs, @victorquinn, @stfuanu, @wut0n9, @Luuhlf, @antonevane, @HappyHelper873, @cckolon, @pkwasniok, @BlazeWits, @timxook, @PangHuJiang, @sjcnyc
